### PR TITLE
🍂 fix(of discussion #4196) Reformulated the theme 'mt'; feature:(add functionality within the theme)

### PR DESCRIPTION
**Pull request by nunniii**
### Prerequisites

- [X] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [X] The commit message follows the [conventional commits][cc] guidelines.
- [x] Tests for the changes have been added (for bug fixes / features).
- [ ] Docs have been added/updated (for bug fixes / features).

image

### 🦄⭐ Description 

just a few changes, including time format, system info (RAM, CPU), cargo and node versions, and command runtime 🚀🪐💮

### CODE REVIEW

**Comment by nunniii** [comment]
sorry I'll correct a mistake before

File: themes/mt.omp.json [hunk 1]
```diff
@@ -16,50 +16,128 @@
...
+          "style": "diamond",
+          "template": " \uf0e7 ",
+          "type": "root"
+        },
+        {
+          "background": "#FFFF00",
+          "foreground": "#000000",
+          "leading_diamond": "\ue0b6",
+          "style": "diamond",
+          "template": " <#000000>🍂 CPU:</> {{ round .PhysicalPercentUsed .Precision }}/1e2 ",
```
Comment (JanDeDobbeleer) [inline themes/mt.omp.json]: This is actually incorrect. We don't have CPU usage information available.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

**Comment by nunniii** [comment]
I believe this solves it if I understood it correctly ... :octocat: 
it can sum up the percentage of physical memory usage (RAMus), right?
```json
{
          "background": "#516BEB",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
-         "template": "\ue266 CPU: {{ round .PhysicalPercentUsed .Precision }}% | ",
+         "template": "\ue266 RAMus: {{ round .PhysicalPercentUsed .Precision }}% | ",
           "type": "sysinfo"

 }
```

**Comment by nunniii** [comment]
Maybe this is better 🦕 
image

File: themes/mt.omp.json [hunk 2]
```diff
@@ -16,50 +16,128 @@
...
+          "style": "diamond",
+          "template": " \uf0e7 ",
+          "type": "root"
+        },
+        {
+          "background": "#FFFF00",
+          "foreground": "#000000",
+          "leading_diamond": "\ue0b6",
+          "style": "diamond",
+          "template": " <#000000>🍂 (RAMus)</> {{ round .PhysicalPercentUsed .Precision }}/1e2 ",
```
Comment (JanDeDobbeleer) [inline themes/mt.omp.json:96]: I would remove this as the below indicates the same but without percentages.

Comment (nunniii) [reply themes/mt.omp.json:96]: 😺🍁 I decided this part is a bit unnecessary, what do u think of that? :

image

**Comment by nunniii** [comment]
🪐❌ I'm closing this PR as I've decided to stick with the original theme, keeping it simple. Some still prefer it as is 😉🚀

**Pull request closed without merging**